src/applications/static-pages/health-care-manage-benefits/secure-messaging-page/components/AuthContent/index.js

```diff
@@ -2,11 +2,26 @@
 import React from 'react';
 // Relative imports.
 import CallToActionWidget from 'platform/site-wide/cta-widget';
+import CernerCallToAction from '../../../components/CernerCallToAction';
 import MoreInfoAboutBenefits from '../../../components/MoreInfoAboutBenefits';
 
+const callToActions = [
+  {
+    deriveHeaderText: facilityNames => `Secure Message ${facilityNames}`,
+    href: '',
+    label: 'Secure Message on My VA Health',
+  },
+  {
+    deriveHeaderText: () =>
+      `Secure Message a provider at another VA Medical Center`,
+    href: '',
```

Comment:
Need the 2 `href` links [here](https://adhoc.invisionapp.com/share/DYVMI5OXEQN#/screens/404880626):

![image](https://user-images.githubusercontent.com/12773166/86204053-c3b6ad80-bb23-11ea-9083-4b43a06c2548.png)

Reply:
**From Cerner for Mann-Grandstaff launch:**

Electronic Health Record Modernization (EHRM) Spokane Mann Grandstaff IOC: Veterans receiving care at VA Medical Centers who have converted to EHRM will have the following functionality:

Send a Secure Message to your health care team/Use Secure Messaging
Link to this in Staging My VA Health: https://ehrm-va-test.patientportal.us.healtheintent.com/pages/messaging/inbox
Link to this in PROD My VA Health: https://patientportal.myhealth.va.gov/pages/messaging/inbox